### PR TITLE
Add project to a technology

### PR DESCRIPTION
**Pull request by ryszardmadej**
Add project to a technology functionality (under Technology > Admin > Change usage in projects)

### CODE REVIEW

File: utils/dbhelper.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ var DBHelper = function () {
  * @param error Function to call on error
  */
 DBHelper.query = function (sql, parameters, done, error) {
-    if (process.env.USE_SSL) {
+    if (process.env.USE_SSL && process.env.USE_SSL.toLowerCase() !== 'false') {
```
Comment (gpimblott) [inline utils/dbhelper.js:17]: This works fine on my Mac (non-ssl) and heroku (ssl), which platform are you using ?

File: database/dbInit.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ var statements = [
     'DROP TABLE IF EXISTS project_tech_link',
     'DROP TABLE IF EXISTS tech_status_link',
     'DROP TABLE IF EXISTS technology_stack_link',
+    'DROP TABLE IF EXISTS technology_project_link',
```
Comment (gpimblott) [inline database/dbInit.js:22]: This duplicates the table project_tech_link above.  Just remove the old table

File: utils/dbhelper.js (hunk 1 above)
Comment (ryszardmadej) [inline utils/dbhelper.js:17]: Odd, that was happening on my Mac - process.env.USE_SSL was resolved to a string "false", and that is auto-casted to bool: true

my process.env entry is: USE_SSL=false

**Comment by gpimblott** [comment]
Good job,  We might need a fast way to add a project to lots of technologies from the admin page.  Should be easy.  This would help initial population, otherwise we will have to  add every project/technology combination separately.

File: utils/dbhelper.js (hunk 1 above)
Comment (gpimblott) [inline utils/dbhelper.js:17]: Not a problem.  Leave it it and I'll check in slow time

**Comment by ryszardmadej** [comment]
Good idea. I think that could be done on a project update/add screen. Either a simple multiselect, or a table with all technologies and the checkbox mechanism that is already in place for bulk removal screen?

File: database/dbInit.js (hunk 2 above)
Comment (ryszardmadej) [inline database/dbInit.js:22]: Somehow did not notice that. Removed

**Pull request merged**